site/src/components/WorkspaceScheduleBanner/WorkspaceScheduleBanner.tsx

```diff
@@ -31,12 +39,32 @@ export const shouldDisplay = (workspace: TypesGen.Workspace): boolean => {
   }
 }
 
-export const WorkspaceScheduleBanner: FC<WorkspaceScheduleBannerProps> = ({ workspace }) => {
+export const WorkspaceScheduleBanner: FC<WorkspaceScheduleBannerProps> = ({ __onExtend, workspace }) => {
```

Comment:
Should we default `__onExtend` to something trivial since it's only used in testing?

Reply:
Removed in https://github.com/coder/coder/commit/0b3abe0f94bff155c337eba87c2d0104402a0c42